src/controllers/verify-ketl.ts

```diff
@@ -45,8 +46,14 @@ export default class VerifyKetlController {
     return Promise.all(attestations)
   }
 
+  @Post('/token')
+  token(@Body({ required: true }) { token, type }: AttestationType & Token) {
+    return signAttestationMessage(type, hexlifyString(token))
+  }
+
   @Post('/email-unique')
-  async sendUniqueEmail(
+  @Version('0.2.2')
```

Comment:
Placed this method above the main so the main one doesn't overwrite it

Reply:
I'm just double checking about what the latest version should be, because there is another route `/faracaster` with `v0.2.3`, but `ketl-app` is using `v0.2.1`. Not sure if this should be `0.2.4` or `0.2.2`